src/extension.ts

```diff
@@ -14,11 +14,14 @@ import { ResumeAllSession } from './ResumeAllSession';
 export { ResumeAllSession } from './ResumeAllSession';
 import { SuspendAllSession } from './SuspendAllSession';
 export { SuspendAllSession } from './SuspendAllSession';
+import { CustomReset } from './CustomReset';
+export { CustomReset } from './CustomReset';
```

Comment:
Appreciate this is replicating what's done above. But haven't understood yet why to export here. Maye other reviewers know this part and can explain to us?